createplace_spec.rb

```diff
@@ -0,0 +1,245 @@
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_phone"]').send_keys "030000000"
+		end
+
+		it 'should enter mobile phone number' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_mobile_phone"]').send_keys "062000000"
+		end
+
+		it 'should enter Fax number' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_fax"]').send_keys "060000000"
+		end
+
+		it 'should enter web adress' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_web"]').send_keys "www.ortakoy.ba"
+		end
+
+		it 'should enter the e-mail' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_email"]').send_keys "ortakoy@gmail.com"
+		end
+
+		it 'should enter Facebook url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_facebook_url"]').send_keys "www.facebook/ortakoy"
+		end
+
+		it 'should enter Twitter url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_twitter_url"]').send_keys "www.twitter.com/ortakoy"
+		end
+
+		it 'should enter instagram url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_instagram_url"]').send_keys "www.instagram.com/ortakoy"
+		end
+
+		it 'should enter Foursqare url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_foursqare_url"]').send_keys "www.foursqare.com/ortakoy"
+		end
+
+		it 'should enter Wikipedia url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_wikipedia_url"]').send_keys "www.wikipedia.com/ortakoy"
+		end
+
+		it 'should enter TripAdvisor url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_tripadvisor_url"]').send_keys "www.tripadvisor.com/ortakoy"
+		end
+
+		it 'should enter Youtube url' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_youtube_url"]').send_keys "www.youtube.com/ortakoy"
+		end
+
+		it 'should enter Instagram hashtag' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_instagram_hashtag"]').send_keys "ortakoy"
+		end
+
+		it 'should enter wi-fi information' do
+			@browser.send_keys :tab
+			@browser.input(:xpath => '//*[@id="poi_has_wifi"]').click
+			@browser.input(:xpath => '//*[@id="poi_wifi_pass"]').send_keys "ortakoy2017"
+			@browser.input(:xpath => '//*[@id="poi_wifi_ssid"]').send_keys "Ortakoy"
+		end
+
+		it 'should provide credit cart payment information' do
+			@browser.send_keys :tab
+			@browser.input(id:"poi_accepts_credit_cards").click
+			@browser.element(:xpath => '//*[@id="chk_credit_card_1"]').click
+		end
+
+		it 'should try to upload a PDF file as a picture' do
+			@browser.file_field.set 'C:\rspec_tutorial\testdata\PDFfile.pdf'
+			@browser.span(class: ["icon-remove", "pull-right"]).click

```

Comment:
File path should not be hardcoded. Please use path in project directory. You can use something like this: pdf_path = File.expand_path("../../testdata/PDFfile.pdf", FILE)